commons/servers/asyncHandler.ts

```diff
@@ -0,0 +1,12 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+
+export function asyncHandler(
```

Comment:
I think we should expose this function through commons `index.ts` file, then we can import it like: 

```
import { asyncHandler } from '@catalyst/commons`
```

Reply:
It is commonly bad to re-export things in Node.js (react may have different conventions tho), it completely nullifies the fact that files and modules are in fact namespaces and may hinder the development flows. I'd avoid them if possible.

If you consider this to be a deal breaker I'll make the change.

Reply:
make sense, is not a deal breaker to me. 

so we probably need to review how `commons` package is exposed as module since it actually has one main/types file (index).

Reply:
I committed a change to make it consistent with the rest of the imports.